recap/static/js/src/recap_edit.js

```diff
@@ -125,6 +131,8 @@ function StudioEditableXBlockMixin(runtime, element) {
 
         // Count all the number of items in the Xblock list
         var counter = 0;
+        var inputHtmlFile= $('#xb-field-edit-html_file');
```

Comment:
Please add a black sapce between the variable name and equal char. @andrey-canon